parlai/agents/image_seq2seq/modules.py

```diff
@@ -96,6 +96,7 @@ def __init__(
         image_encoder_num_layers=1,
         image_features_dim=2048,
         image_combination_mode='append',
+        n_image_tokens=1,
```

Comment:
What do you use in practice? This looks like it could cause that multiple-of-8 issue I was mentioning

Reply:
So, my time steps definitely aren't multiples of 8. Given what you said, I'm sure that's causing a major slowdown - I first have to address some issues with the training *itself* not going as expected, but then I'll take a look at this :)

Reply:
if we don't end up seeing any gain from varying this I would say we can just keep it at 1 and reduce some unnecessary complexity later

Reply:
Hmm - so, preliminary sweeps show that it might be giving us a few-point boost, presumably due to the image features just taking up more space in the final encoding. I'd prefer to keep it in unless you all really hate it :)